src/astbuilder.nit

```diff
@@ -157,6 +157,14 @@ class ASTBuilder
 	do
 		return new AReturnExpr.make(expr)
 	end
+	# Create a callsite with the given `mproperty`. Take the current method `actual_method` as a context
+	fun create_callsite(modelbuilder: ModelBuilder, actual_method : AMethPropdef, mproperty: MMethod, is_self_call: Bool): CallSite
```

Comment:
Documentation not clear. Is `actual_method` the caller and `mproperty` the callee ?